pkg/management/endpoint_test.go

```diff
@@ -1150,6 +1151,18 @@ func TestSendOnboardingEmailEndpoint(t *testing.T) {
 		assert.Nil(t, err)
 		assert.Nil(t, res)
 	})
+	t.Run("Valid context-key parameter", func(t *testing.T) {
+		delete(req, prmQryLifespan)
```

Comment:
If you don't like how req is populated, you'd rather define a new map with your own values